FrameworkDesignConsiderations.md

```diff
@@ -28,7 +28,7 @@ follow the same pattern: read data and then transform it new data, and then make
 data available in the TES. As such, one could consider writing one single generic algorithm
 which wraps this functionality (and thus provides a uniform pattern), and delegate the
 computational work. This suggests a new type of Gaudi component, which is explictly provided
-with input data, and explicitly provides output data -- lets define this as a Transform.. 
+with input data, and explicitly provides output data -- let's define this as a Transform.
 Note that there is no direct interaction between these Transform components and the TES -- this
```

Comment:
Yes ;-) The above defines the communication 'within' a process, and we need a way to communicate beyond the bounds of a process, i.e. persist the output. And in the end, all of these are 'key-value' pairs, and whether the key is a 'label' in the TES or an instance of an algorithm doesn't really matter...